dark/reads.py

```diff
@@ -1512,6 +1512,30 @@ def variableSites(self, confirm=False, homogeneityLevel=1.0,
 
         return varSites
 
+    def combineReads(self):
+        """
+        Combine all reads into a single read.
+        """
+        reads = list(self)
+        length = len(reads[0].sequence)
+        sequence = ''
+        for site in range(length):
+            bases = set([r.sequence[site] for r in reads])
+            if len(bases) == 1:
+                sequence += list(bases)[0]
```

Comment:
`bases.pop()`

Reply:
I think this section of code can be replaced with the following:

```python
from itertools import zip_longest

for bases in zip_longest(*(read.sequence for read in reads), fillvalue='-'):
    bases = set(bases)
    # ...
```

And you can add some tests for uneven sequences :-)